### ISSUE TITLE
menu : expand/collapse menu items without navigation

### ISSUE THREAD

**Issue by kadavil** [McShelby/hugo-theme-relearn#231]
Hello, 

I have a feature request with the Menu.  I am looking for a way to collapse a menu item on a click on the menu item. 
Currently when you click on a menu item, it expands and displays the sub menu items, but clicking on the main menu item does not collapse it.  I have to click on another menu item to collapse the open one.

Example:  (here is a sample menu with sub menu items.

Welcome
  - Team
  - Engineers
  - Product Owners
Onboarding
  - Access Requests
  - Laptop Setup
  - Service Now 
Testing
  - TDD
  - Go
  - React
  
Initially only these 3 are display.
Welcome
Onboarding
Testing

On clicking on Onboarding, the sub menus are displayed
Welcome
Onboarding
  - Access Requests
  - Laptop Setup
  - Service Now 
Testing

To collapse the Onboarding, I need to click on either Welcome or Testing, which collapses the Onboarding menu, but  expands the one I click.

Is it possible to make the expansion/collapse work like a toggle? A click to expand, and next click on the top menu item to collapse?

**Comment by McShelby** [comment]
I did this earlier (before forking the theme) in a hacked local installation.

Nevertheless the implementation was mediocre, because I did not change the DOM from inside a Hugo template but did run a JavaScript function to apply runtime changes to the DOM after load.

So, now that I forked the whole project, this is absolutly doable in a more performant way and sounds like a good idea. There are some tricky details, especially "available space". So I have to think it thru.
